cmd/stayrtr/stayrtr.go

```diff
@@ -517,13 +517,13 @@ func (s *state) updateFile(file string) (bool, error) {
 	}
 
 	log.Infof("new cache file: Updating sha256 hash %x -> %x", s.lasthashCache, hsum)
```

Comment:
Please change this log.Infof to log.Debugf this message is now really not needed for normal operation.